### DISCUSSION TITLE
怎么获取更详细的文档

### DISCUSSION THREAD

**Discussion by rien7** [unocss/unocss#3285] [General]
例如在 `preset-mini` 中的 [border](https://github.com/unocss/unocss/blob/main/packages/preset-mini/src/_rules/border.ts) 可以使用 `rounded` 或者 `rd` 来设置边框圆角。前者容易通过 [TailwindCSS 的文档](https://www.tailwindcss.cn/docs/border-radius) **猜**到，后者则需要查看具体的正则表达式才能知道其缩写。

是否有可能将规则以及对应生成的 CSS 用**更易用**的方式展示，就像 [TailwindCSS](https://www.tailwindcss.cn/docs/installation) 一样。目前的 [Interactive Docs](https://unocss.dev/interactive/) 并不是一个好的查询文档。

有什么思路去实现这一想法吗？

**Comment by elarbor** [comment]
现在更缺中文文档

**Reply by zyyv** [reply]
Hi~, 中文文档开发和维护，不是我们的方向，还请看看社区是否有人维护同步中文文档，我们愿意提供链接到它。
没有确实有点困惑，有使用方式的疑问请参阅 [Tailwind 文档](https://tailwindcss.com/docs), 我们完全兼容其用法，我们的文档更多的在于对其每个包和预设的介绍和使用。
